### ISSUE TITLE
Implement .style(style_obj) operation!

### ISSUE THREAD

**Issue by Olian04** [Olian04/brynja#3]
### Finished design for v1.0

Since the types for a CSS-IN-JS object is HUGE, I've decided to use the auto generated types of [csstype](https://github.com/frenic/csstype). Same types as React uses.

```ts
import { render } from 'brynja';

render(_=>_
  .style({
    border: {
      style: 'solid', // border-style: solid;
      color: 'black', // border-color: black;
      width: '2px', // border-width: 2px;
    },
    border: 'solid black 2px', // border: solid black 2px;
    ':active': {
      color: 'skyblue',
    },
    '::before': {
      content: '--> ',
    },
    '::after': {
      content: ' <--',
    },
  })
);
```

**Comment by Olian04** [comment]
Could borrow logic from https://github.com/Olian04/styled-vanilla

**Comment by Olian04** [comment]
Basic:

```js
import { render } from 'brynja';

render(_=>_
  .style({
     background: 'orange'
  })
);
```

Alternative syntax: 

```js
import { render } from 'brynja';

render(_=>_
  .style({
    border: {
      style: 'solid', // border-style: solid;
      color: 'black', // border-color: black;
      width: '2px', // border-width: 2px;
    },
    border: 'solid black 2px', // border: solid black 2px;
  })
);
```

Pseudo selectors:

```js
import { render } from 'brynja';

render(_=>_
  .style({
    $active: {
      color: 'skyblue',
    },
    $before: {
      content: '--> ',
    },
    $after: {
      content: ' <--',
    },
  })
);
```

**Comment by Olian04** [comment]
This has been implemented and published in the 3.0.0 release
